### PR TITLE
[RHACS] Fixed `roxctl` installation gaps

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-15094 
Updates to the existing docs based on internal technical review.

Cherrypick into:
- `rhacs-docs-3.74`

Preview: 
- Site: https://55791--docspreview.netlify.app/openshift-acs/latest/welcome/index.html
- General requirements: https://55791--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/prerequisites-other.html#acs-general-requirements_prerequisites-other
- Installing Sensor by using the generate YAML files: https://55791--docspreview.netlify.app/openshift-acs/latest/cli/getting-started-cli.html#installing-sensor-by-using-the-generate-yaml-files
- Installing the central-services Helm chart without customizations: https://55791--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/install-central-other.html#installing-quickly_install-central-other

### CODE REVIEW

File: modules/install-secured-cluster-services-helm-chart.adoc [hunk 1]
```diff
@@ -12,6 +12,15 @@ After you configure the `values-public.yaml` and `values-private.yaml` files, in
 To install Collector on systems that have Unified Extensible Firmware Interface (UEFI) and that have Secure Boot enabled, you must use eBPF probes because kernel modules are unsigned, and the UEFI firmware cannot load unsigned packages. Collector identifies Secure Boot status at the start and switches to eBPF probes if required.
 ====
 
+.Prerequisites
```
Comment (bmcelvee) [inline modules/install-secured-cluster-services-helm-chart.adoc:15]: I'm struggling to find this is the preview, but content-wise it's good!

**Comment by bmcelvee** [comment]
One note about a bit of content I couldn't find in the preview. Otherwise, LGTM!

File: installing/acs-installation-platforms.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-{rh-rhacs-first} and {product-title-managed} ({product-title-managed-short}) are supported on{ocp} and Kubernetes platforms. 
+{rh-rhacs-first} and {product-title-managed} ({product-title-managed-short}) are supported on{ocp} and Kubernetes platforms.
```
Comment (kowen-rh) [inline installing/acs-installation-platforms.adoc:9]: I think this is a typo: `on{ocp}`

**Comment by gaurav-nelson** [comment]
Thank you @bmcelvee I missed that one in the description. The preview for that section appears at https://55791--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/install-secured-cluster-other.html#installing-secured-cluster-services-quickly_install-secured-cluster-other

File: installing/acs-installation-platforms.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply installing/acs-installation-platforms.adoc:9]: @kowen-rh thank you.

File: modules/acs-quick-install-secured-cluster-using-helm.adoc [hunk 3]
```diff
@@ -22,6 +22,7 @@ To install Collector on systems that have Unified Extensible Firmware Interface
 ====
 
 .Prerequisites
+* You must have generated {product-title-short} init bundle for your cluster.
```
Comment (kowen-rh) [inline modules/acs-quick-install-secured-cluster-using-helm.adoc:25]: Just a note that "have generated" is passive voice. I'm not sure that active voice is necessary here, but thought it was worth mentioning. :slightly_smiling_face:

**Pull request merged**